.github/workflows/build-and-test-windows.yml

```diff
@@ -89,3 +89,40 @@ jobs:
             echo "One or more matrix jobs failed."
             false
           fi
+  integration-tests:
+    if: ${{ github.actor != 'dependatbot'[bot] && (contains(github.event.pull_request.labels.*.name, 'Run Windows') || github.event_name == 'push') }}
+    runs-on: windows-latest
+    steps:
+      - name: Collect Workflow Telemetry
+        if: always()
+        uses: runforesight/foresight-workflow-kit-action@v1
```

Comment:
We are removing foresight as part of another PR.

Reply:
I updated the workflow, it still runs integration tests on a Windows machine, but it no longer collects workflow telemetry or analyzes test and coverage results using Foresight